Sources/Public/CombineExtension.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  CombineExtension.swift
+//  SwiftPrettyPrint
+//
+//  Created by Yusuke Hosonuma on 2020/11/26.
+//
+
+// Linux is not supported to Combine framework.
+#if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
+
+    import Combine
+
+    @available(iOS 13.0, *)
+    @available(OSX 10.15, *)
```

Comment:
In line 9, we have a condition for `watchOS` and `tvOS`.
Therefore, I think the `@available` annotation should include those OS as follows:

```suggestion
    @available(macOS 10.15, iOS 13.0, watchOS 6, tvOS 13, *)
```

Reply:
You're right. Thanks!